doc/source/ray-core/handling-dependencies.rst

```diff
@@ -266,6 +266,7 @@ However, using runtime environments you can dynamically specify packages to be a
 
 
 You may also specify your ``pip`` dependencies either via a Python list or a local ``requirements.txt`` file.
+Specifying a ``requirements.txt`` file is useful when your ``pip install`` command would require options such as ``--extra-index-url`` or ``--find-links``; see `<https://pip.pypa.io/en/stable/reference/requirements-file-format/#>`_ for details.
```

Comment:
```suggestion
Consider specifying a ``requirements.txt`` file when your ``pip install`` command requires options such as ``--extra-index-url`` or ``--find-links``; see `<https://pip.pypa.io/en/stable/reference/requirements-file-format/#>`_ for details.
```

Reply:
If it's not obvious why it might be useful, maybe you can add a reason, briefly.